specs/CameraCaptureUI/CameraCaptureUI.md

```diff
@@ -0,0 +1,382 @@
+CameraCaptureUI API
+===
+
+This is the spec for proposal: [Issue #1034](https://github.com/microsoft/WindowsAppSDK/issues/1034)
+
+This spec details the API of a new `CameraCaptureUI` in WinAppSDK,
+enabling desktop applications such as WinUI3 to seamlessly integrate native camera capture 
+functionality across diverse Windows platforms.
+
+
+
+# Background
+
+The current OS 
+[CameraCaptureUI](https://docs.microsoft.com/uwp/api/Windows.Media.Capture.CameraCaptureUI),
+primarily designed for UWP applications,
+presents several challenges when used in desktop environments. 
+Key issues include the dependency on `CoreWindow`, 
+which are not compatible with desktop app frameworks and the lack of support for `IInitializeWithWindow`. 
+As a result, developers are forced to resort to workarounds involving interop interfaces and 
+additional code to integrate camera capture functionality into their WinUI 3 desktop applications.
+
+In response to this feedback, this `CameraCaptureUI` API for WinAppSDK aims to provide 
+a streamlined solution that meets the expectations of our developer community. 
+The goal is to offer seamless integration of camera capture capabilities with 
+full feature parity across all Windows platforms supported by WinAppSDK. 
+This new API eliminates the need for cumbersome workarounds and 
+simplifies the process of incorporating camera functionality into desktop applications.
+
+
+This new WinAppSDK version of the `CameraCaptureUI` is essentially the same as the existing public
+[CameraCaptureUI](https://learn.microsoft.com/en-us/uwp/api/windows.media.capture.cameracaptureui?view=winrt-22621) 
+API, with a minor adjustment to the CameraCapture constructor to require a
+[WindowId](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.WindowId).
+
+
+
+
+
+
+# Conceptual pages
+
+## Capture a photo (C++)
+
+A C++ example that tests whether a photo was successfully taken using the CaptureFileAsync method.
+
+```c++
+    // Initialize CameraCaptureUI with a window handle (Foreground Window in this example)
+    auto parentWindow = ::GetForegroundWindow();
+    winrt::Microsoft::UI::WindowId windowId{ reinterpret_cast<uint64_t>(parentWindow) };
+    winrt::Microsoft::Windows::Media::Capture::CameraCaptureUI cameraUI(windowId);
+
+    // Configure Photo Settings
+    cameraUI.PhotoSettings().Format(CameraCaptureUIPhotoFormat::Jpeg);
+    cameraUI.PhotoSettings().AllowCropping(false);
+
+    // Capture a photo asynchronously
+    auto photoOperation = cameraUI.CaptureFileAsync(CameraCaptureUIMode::Photo);
+    auto photo = photoOperation.get(); // Get the result synchronously
```

Comment:
Usually the C++ & C# code samples should match. In this case C# handles this async and C++ does this sync. Should they both be sync or both be async?

Reply:
Agreed, updated the C++ sample to call async. Thanks!